server/core/src/https/views/login.rs

```diff
@@ -835,10 +835,8 @@ async fn view_login_step(
                 break res;
             }
             AuthState::Continue(allowed) => {
-                // Reauth inits its session here so we need to be able to add cookie here ig.
-                if jar.get(COOKIE_AUTH_SESSION_ID).is_none() {
-                    jar = add_session_cookie(&state, jar, &session_context)?;
-                }
+                // Reauth inits its session here so we need to be able to add it's cookie here.
+                jar = add_session_cookie(&state, jar, &session_context)?;
```

Comment:
`add_session_cookie` sets `SameSite::Strict`, do we really want to keep using that?

Reply:
Yes. This is the auth-session-id, so you'd only be getting it within the samesite.

Like if you navigate away then back to Kanidm, you should get a new auth-session.